test/test_kmod/mod.rs

```diff
@@ -0,0 +1,93 @@
+use std::process::Command;
+
+fn compile_kernel_module() -> String {
+    #[allow(unused_variables)]
```

Comment:
The `#[allow(unused_variables)]` syntax is holdover from an older version of rustc.  Nowadays you can just do `let _, = ...` and it won't complain.

Reply:
Sorry, my last comment had a typo.  You need to use `let _m = ...`, not `let _ = ...`.  There is a difference between `_m` and `_`.

Reply:
Good to know. I didn't know they behave differently. I update the code.